packages/itwinui-css/src/button/button.scss

```diff
@@ -33,42 +33,35 @@
   }
 
   &[data-iui-active='true'] {
-    @include default.iui-button-default-active;
+    @include active.iui-button-active;
+    @include active.iui-button-default-active;
 
-    &[data-iui-variant='borderless'] {
-      @include borderless.iui-button-borderless-active;
+    &:is([data-iui-variant='borderless']) {
+      @include active.iui-button-borderless-active;
     }
-
-    @include default.iui-button-active-disabled;
   }
 
-  &[data-iui-variant='high-visibility'] {
-    @include variant.iui-button-variant(high-visibility);
-  }
+  @include variant.iui-button-variant;
 
-  &[data-iui-variant='cta'] {
-    @include variant.iui-button-variant(cta);
-  }
-
-  &[data-iui-variant='idea'] {
+  &[data-iui-type='feedback'] {
```

Comment:
I am wondering if this would make more sense to keep as a value on `data-iui-variant` instead. After all, the PR says that the ideas button is going to need to have a specific variant value anyways in order to be colored correctly. If both `high-visibility` and `cta` ideas buttons are meant to be seen as correctly colored, then that would make this change make sense. However, our `css-workshop` page for buttons only shows `high-visibility` coloring. Adding a new example with alternate coloring could make that intention more clear.

Reply:
If we keep the ideas button as a `data-iui-variant` that would mean the CSS for the coloring would need to be duplicated.

If we go this new route, the CSS is less but there is more HTML to define both type and variant.  I don't have anyone requesting a green ideas button, but this new method does allow it if the need arrises.  I think I would lean towards the new method due to less duplicate CSS and more flexibility, but would like to hear what others think as well.

Reply:
Okay, I think that I agree that reducing the duplicate CSS should be the main focus after thinking about it a bit more.